network-protection/network-protection-impl/src/main/res/values/strings-netp.xml

```diff
@@ -235,8 +235,9 @@
     <string name="netpDdgDnsByLine">DuckDuckGo routes DNS queries through our DNS servers so your internet provider can\'t see what websites you visit.</string>
 
     <!-- Block malware DNS -->
-    <string name="netpDnsBlockMalwarePrimary">Block Malware</string>
-    <string name="netpDnsBlockMalwareByline">Block malware with a DNS-level blocklist. If a website doesn\'t load, try turning blocking off.</string>
+    <string name="netpDnsBlockMalwareHeader">Block lists</string>
```

Comment:
I’ll request translations once we have approved copy (in ship review) since we haven’t had any copy task prior so things could likely change here.